File: server/dotnet/HostedCheckout/CreateHostedCheckoutBasicDto.cs
namespace OnlinePayments.Sdk.Example;

/// <summary>
/// Basic Dto for hosted checkout used for testing purposes
/// </summary>
public class CreateHostedCheckoutBasicDto
{
    public decimal Amount { get; set; }

    public string? Currency { get; set; }

    public string? RedirectUrl { get; set; }

}

File: server/java/src/main/java/com/worldlinesolutions/onlinepayments/example/hostedcheckout/CreateHostedCheckoutBasicDto.java
package com.worldlinesolutions.onlinepayments.example.hostedcheckout;

import java.math.BigDecimal;

/**
 * Basic Dto for hosted checkout used for testing purposes
 */
public class CreateHostedCheckoutBasicDto {

    private BigDecimal amount;

    private String currency;

    private String redirectUrl;

    public BigDecimal getAmount() {
        return amount;
    }

    public void setAmount(BigDecimal amount) {
        this.amount = amount;
    }

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }

    public String getRedirectUrl() {
        return redirectUrl;
    }

    public void setRedirectUrl(String redirectUrl) {
        this.redirectUrl = redirectUrl;
    }
}
